model/src/main/java/org/mskcc/cmo/metadb/model/PatientMetadata.java

```diff
@@ -26,7 +26,7 @@
     @Relationship(type = "PX_TO_SP", direction = Relationship.OUTGOING)
     private List<SampleManifestEntity> sampleManifestList;
     @Relationship(type = "PX_TO_NORMAL", direction = Relationship.OUTGOING)
-    private List<SampleManifestEntity> normalSampleManifestList;
+    private List<NormalSampleEntity> normalSampleManifestList;
```

Comment:
will need to update the type here once the normal sample manifest entity class is renamed